src/StepperMotorReadback.cc

```diff
@@ -7,118 +7,175 @@
 
 #include "StepperMotorReadback.h"
 #include "ChimeraTK/MotorDriverCard/StepperMotorException.h"
+#include <mtca4u/MotorDriverCard/MotorDriverException.h>
 
 namespace ChimeraTK { namespace MotorDriver {
 
   void ReadbackHandler::mainLoop() {
     readConstData();
+    unsigned int spiErrorCounter = 0;
+    const unsigned int SPI_RETRY_COUNT = 50;
```

Comment:
Isn't 50 re-tries a bit high? If that thing produces 50 errors before it succeeds it's pretty much useless. How do we get a stop signal for the motor through like that?

Reply:
Probably. This was based on the fact that the usual trigger for that module was supposed to be the MacroPulse, so that is about 5 seconds.

Reply:
Same as with the mtca4u include. MotorDriverCard needs adaption. https://redmine.msktools.desy.de/issues/9678

Reply:
Changed to 5 tries. I have no idea how frequent or persistent those are since the first one currently kills off the server